packages/core/admin/ee/server/services/review-workflows/review-workflows.js

```diff
@@ -106,21 +106,43 @@ function enableReviewWorkflow({ strapi }) {
       const { joinTable } = strapi.db.metadata.get(target).attributes[morphBy];
       const { idColumn, typeColumn } = joinTable.morphColumn;
 
-      // Execute a raw SQL query to insert records into the join table mapping the specified content type with the first stage of the default workflow.
+      // Execute an SQL query to insert records into the join table mapping the specified content type with the first stage of the default workflow.
       // Only entities that do not have a record in the join table yet are selected.
-      await strapi.db.connection
-        .raw(`INSERT INTO ${joinTable.name} (${idColumn.name}, field, "order", ${joinTable.joinColumn.name}, ${typeColumn.name})
-                SELECT
-                    entity.id as ${idColumn.name},
-                    '${ENTITY_STAGE_ATTRIBUTE}' as field,
-                    1 as "order",
-                    ${firstStage.id} as ${joinTable.joinColumn.name},
-                    '${contentTypeUID}' as ${typeColumn.name}
-                FROM ${contentTypeMetadata.tableName} entity
-                LEFT JOIN ${joinTable.name} jointable
-                ON  entity.id = jointable.${idColumn.name}
-                AND jointable.${typeColumn.name} = '${contentTypeUID}'
-                WHERE jointable.${idColumn.name} IS NULL`);
+      const connection = strapi.db.connection;
+      const columnsToInsert = [
+        idColumn.name,
+        'field',
+        `"order"`,
+        joinTable.joinColumn.name,
+        typeColumn.name,
+      ];
+
+      const selectStatement = connection
+        .select(
+          `entity.id as ${idColumn.name}`,
+          connection.raw(`'${ENTITY_STAGE_ATTRIBUTE}' as field`),
+          connection.raw(`1 as "order"`),
+          connection.raw(`'${firstStage.id}' as ${joinTable.joinColumn.name}`),
+          connection.raw(`'${contentTypeUID}' as ${typeColumn.name}`)
+        )
+        .leftJoin(`${joinTable.name} AS jointable`, function () {
+          this.on('entity.id', '=', `jointable.${idColumn.name}`).andOn(
+            `jointable.${typeColumn.name}`,
+            '=',
+            connection.raw(`'${contentTypeUID}'`)
+          );
+        })
+        .where(`jointable.${idColumn.name}`, null)
+        .from(`${contentTypeMetadata.tableName} AS entity`)
+        .toSQL();
```

Comment:
maybe we could use the identifier syntax from knex https://knexjs.org/guide/query-builder.html#identifier-syntax
something like 
```suggestion
        .select({
          idColumn.name: entity.id,
          field: ENTITY_STAGE_ATTRIBUTE,
					...
        )
        .leftJoin(`${joinTable.name} AS jointable`, function () {
          this.on('entity.id', '=', `jointable.${idColumn.name}`).andOn(
            `jointable.${typeColumn.name}`,
            '=',
            connection.raw(`'${contentTypeUID}'`)
          );
        })
        .where(`jointable.${idColumn.name}`, null)
        .from(`${contentTypeMetadata.tableName} AS entity`)
        .toSQL();
```

Reply:
I've refactored to use the identifier syntax as much as I could. However, I couldn't find a way to change the select clause to identifier syntax as we are selecting string literals rather than column names

Reply:
Was playing with this just now, I think this is working. Do you think it could be enough?
```js
      const selectStatement = connection
        .select({
          [idColumn.name]: 'entity.id',
          field: strapi.db.connection.raw('?', [ENTITY_STAGE_ATTRIBUTE]),
          order: 1,
          [joinTable.joinColumn.name]: firstStage.id,
          [typeColumn.name]: connection.raw('?', [contentTypeUID]),
        })
        .leftJoin(`${joinTable.name} AS jointable`, function () {
          this.on('entity.id', '=', `jointable.${idColumn.name}`).andOn(
            `jointable.${typeColumn.name}`,
            '=',
            connection.raw('?', [contentTypeUID])
          );
        })
        .where(`jointable.${idColumn.name}`, null)
        .from(`${contentTypeMetadata.tableName} AS entity`)
        .toSQL();

      // Insert the clone relations
      await connection(joinTable.name).insert(
        connection.raw(
          `(${columnsToInsert.join(',')})  ${selectStatement.sql}`,
          selectStatement.bindings
        )
      );
      ```

Reply:
btwfor the entity_stage_attribute and the contentTypeUID I think is better to use the connection.raw('?', [bindings]) format, as knex automatically adds the `'`  if necessary